cli/src/pcluster/config/cluster_config.py

```diff
@@ -1555,10 +1567,6 @@ def __init__(self, cluster_name: str, scheduling: AwsBatchScheduling, **kwargs):
     def _register_validators(self):
         super()._register_validators()
         self._register_validator(AwsBatchRegionValidator, region=self.region)
```

Comment:
This kind of change of moving `SchedulerOsValidator` and `HeadNodeImdsValidator` from AwsBatchClusterConfig to BaseClusterConfig should stay in a separate commit.

Reply:
I split the PR in to two commits